library/core/src/intrinsics/mir.rs

```diff
@@ -275,7 +276,7 @@ define!("mir_return", fn Return() -> BasicBlock);
 define!("mir_goto", fn Goto(destination: BasicBlock) -> BasicBlock);
 define!("mir_unreachable", fn Unreachable() -> BasicBlock);
 define!("mir_drop", fn Drop<T>(place: T, goto: BasicBlock));
-define!("mir_call", fn Call<T>(place: T, goto: BasicBlock, call: T));
+define!("mir_call", fn Call<T>(call: T, goto: BasicBlock));
```

Comment:
The type of the first argument should be `()` I believe

Reply:
Ah right, I forgot that assignments have type `()` in Rust.